src/append/file.rs

```diff
@@ -89,26 +90,56 @@ impl FileAppenderBuilder {
     /// will be resolved. Note that if the variable fails to resolve,
     /// $ENV{name_here} will NOT be replaced in the path.
     pub fn build<P: AsRef<Path>>(self, path: P) -> io::Result<FileAppender> {
```

Comment:
Update this comment to reference your TIME option as well.

Reply:
Thanks for you review. I updated this comment including `$ENV{name_here}` and `$TIME{chrono_format}`